src/library/export/trackexportworker.h

```diff
@@ -63,6 +66,7 @@ class TrackExportWorker : public QThread {
             const QString& filename,
             std::promise<TrackExportWorker::OverwriteAnswer>* promise);
     void progress(const QString& filename, int progress, int count);
+    void missing();
```

Comment:
I don't understand what this signal is trying to tell me? Please improve the naming.

Reply:
this should have been deleted. will fix